src/pages/AnimeOnegai/main.ts

```diff
@@ -0,0 +1,140 @@
+import { pageInterface } from '../pageInterface';
+
+let id;
+let OVurl;
+let Interval;
+
+export const AnimeOnegai: pageInterface = {
+  name: 'AnimeOnegai',
+  domain: 'https://www.animeonegai.com',
+  languages: ['Spanish'],
+  type: 'anime',
+  isSyncPage(url) {
+    if (utils.urlPart(url, 4) === 'watch') {
+      if (j.$('div.backButtonAssetInfo > h2').text().trim() !== 'Canal Onegai') return true;
+    }
+
+    return false;
+  },
+  isOverviewPage(url) {
+    if (utils.urlPart(url, 4) === 'details') return true;
+
+    return false;
+  },
+  getImage() {
+    return $('div > img.img-poster').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('div.backButtonAssetInfo > div > small').text().trim();
+    },
+    getIdentifier(url) {
+      return id;
+    },
+    getOverviewUrl(url) {
+      return OVurl;
+    },
+    getEpisode(url) {
```

Comment:
Better to use regex group matching for this as it is easier and more reliable
Example : https://github.com/ijcs/malsync/blob/0d7efdf8bdc4bb35357149afc8e696f5dce48bde/src/pages/ADN/main.ts#L25

Reply:
I already changed it to an alternative like the one in the example... but gh did not detect it. Notice manually.